cmi5-profile.jsonld

```diff
@@ -0,0 +1,517 @@
+            "sequence": ["_:launched", "_:initialized", "_:failed", "_:terminatedorabandoned"]
+        },
+        {
+            "id": "_:completionmaybefailedsession",
+            "inScheme": "https://w3id.org/xapi/cmi5/context/categories/cmi5",
+            "sequence": ["_:launched", "_:initialized", "_:completedandmaybefailed", "_:satisfieds", "_:terminatedorabandoned"]
+        },
+        {
+            "id": "_:terminatedorabandoned",
+            "inScheme": "https://w3id.org/xapi/cmi5/context/categories/cmi5",
+            "alternates": ["_:terminated", "_:abandoned"]
+        },
+        {
+            "id": "_:completedandpassed",
+            "inScheme": "https://w3id.org/xapi/cmi5/context/categories/cmi5",
+            "alternates": ["_:completedthenpassed", "_:passedthencompleted"]
+        },
+        {
+            "id": "_:completedthenpassed",
+            "inScheme": "https://w3id.org/xapi/cmi5/context/categories/cmi5",
+            "sequence": ["_:completed", "_:satisfieds", "_:passed"]
+        },
+        {
+            "id": "_:passedthencompleted",
+            "inScheme": "https://w3id.org/xapi/cmi5/context/categories/cmi5",
+            "sequence": ["_:passed", "_:satisfieds", "_:completed"]
+        },
+        {
+            "id": "_:completedandmaybefailed",
+            "inScheme": "https://w3id.org/xapi/cmi5/context/categories/cmi5",
+            "alternates": ["_:maybecompletedthenfailed", "_:failedthenmaybecompleted"]
+        },
+        {
+            "id": "_:maybecompletedthenfailed",
+            "inScheme": "https://w3id.org/xapi/cmi5/context/categories/cmi5",
+            "sequence": ["_:maybecompleted", "_:satisfieds", "_:failed"]
+        },
+        {
+            "id": "_:failedthenmaybecompleted",
+            "inScheme": "https://w3id.org/xapi/cmi5/context/categories/cmi5",
+            "sequence": ["_:failed", "_:maybecompleted"]
+        },
+        {
+            "id": "_:maybecompleted",
+            "inScheme": "https://w3id.org/xapi/cmi5/context/categories/cmi5",
+            "optional": "_:completed"
+        },
+        {
+            "id": "_:typicalsession",
+            "inScheme": "https://w3id.org/xapi/cmi5/context/categories/cmi5",
+            "alternates": [
+                "_:completionmaybefailedsession",
+                "_:completionpassedsession",
+                "_:failedsession",
+                "_:noresultsession",
+                "_:passedsession",
+                "_:completionnosuccesssession",
+                "_:waivedsession"
+            ]
+        },
+        {
+            "id": "_:typicalsessions",
+            "inScheme": "https://w3id.org/xapi/cmi5/context/categories/cmi5",
+            "zeroOrMore": "_:typicalsession"
+        },
+        {
+            "id": "_:toplevel",
+            "inScheme": "https://w3id.org/xapi/cmi5/context/categories/cmi5",
+            "sequence": ["_:satisfieds", "typicalsessions"]
```

Comment:
I wonder if there any reason for why the last pattern with id `"_:toplevel"` is referencing `"typicalsessions"`  instead of `"_:typicalsessions"` (starting with the **underscore and colon**, such as all the other references). But, I don't know much about this format yet so perhaps it is not a problem. It looks very interesting.

(I am new to Github so sorry if I am using it in a wrong way).